src/main/java/dto/BuyerOrder.java

```diff
@@ -0,0 +1,10 @@
+package dto;
+
+public class BuyerOrder extends AbstractOrder {
```

Comment:
I think if we will have `orderType` field in Order class, we won't need these

Reply:
And if we have different income parameters for each order type? Like buyer names to calculate some discount for regular users? I guess we need each of them. Do you agree or I just remove?